### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by oxyc**
Just to get things working a bit better I'm including some stray changes I have locally.

The plugins we still need to revamp a bit I guess and I have a few I want to add but haven't. We can revisit them later. For now I'm adding a few debugging plugins I've needed and are useful to have in the codebase when needed both on production and local.

One notable change here is that im requiring the Gutenberg plugin so we get all the latest features.

Other than that I'm fixing slow NFS syncs that break livereload. These could require more research but for now these have been working quite well. I have some more notes in the commit message.

### CODE REVIEW

**Comment by toffebjorkskog** [comment]
> Other than that I'm fixing slow NFS syncs that break livereload.

Hurraay!

File: composer.json [hunk 1]
```diff
@@ -45,6 +45,7 @@
     "oscarotero/env": "^1.1.0",
     "roots/wp-password-bcrypt": "1.0.0",
     "roots/wp-config": "1.0.0",
+    "jjgrainger/posttypes": "~2.0",
```
Comment (toffebjorkskog) [inline composer.json:48]: https://posttypes.jjgrainger.co.uk/

File: composer.json [hunk 2]
```diff
@@ -59,9 +60,9 @@
     "wpackagist-plugin/post-types-order": "~1.9",
     "wpackagist-plugin/taxonomy-terms-order": "^1.5",
     "wpackagist-plugin/wp-smartcrop": "~2.0",
-    "roots/multisite-url-fixer": "^1.1",
 
     "generoi/gravityforms": "~2.1",
+    "generoi/gravityformsgutenberg": "~1.0-rc",
```
Comment (toffebjorkskog) [inline composer.json:65]: https://github.com/generoi/gravityformsgutenberg

File: composer.json [hunk 3]
```diff
@@ -85,19 +85,22 @@
     "mcguffin/acf-quick-edit-fields": "^2.4",
     "wpackagist-plugin/show-environment-in-admin-bar": "^1.1",
     "soberwp/intervention": "^1.2",
+    "wpackagist-plugin/regenerate-thumbnails": "~3.0",
     "wpackagist-plugin/debug-bar": "^1.0",
+    "wpackagist-plugin/debug-bar-remote-requests": "^0.1.2",
```
Comment (toffebjorkskog) [inline composer.json:90]: What is this for

File: composer.json [hunk 4]
```diff
@@ -85,19 +85,22 @@
...
     "wpackagist-plugin/kint-debugger": "~1.0",
-    "wpackagist-plugin/regenerate-thumbnails": "~3.0",
+    "wpackagist-plugin/query-monitor": "^3.3",
```
Comment (toffebjorkskog) [inline composer.json:92]: https://querymonitor.com/

> Query Monitor is the developer tools panel for WordPress. It enables debugging of database queries, PHP errors, hooks and actions, block editor blocks, enqueued scripts and stylesheets, HTTP API calls, and more.

File: composer.json [hunk 5]
```diff
@@ -85,19 +85,22 @@
...
+    "wpackagist-plugin/user-switching": "^1.5",
```
Comment (toffebjorkskog) [inline composer.json:93]: https://wordpress.org/plugins/user-switching/

> This plugin allows you to quickly swap between user accounts in WordPress at the click of a button. You’ll be instantly logged out and logged in as your desired user. This is handy for test environments where you regularly log out and in between different accounts, or for administrators who need to switch between multiple accounts.

File: composer.json [hunk 6]
```diff
@@ -85,19 +85,22 @@
...
     "wp-cli/wp-cli-bundle": "^2.2",
     "cweagans/composer-patches": "~1.0",
-    "jjgrainger/posttypes": "~2.0",
     "generoi/genero-conf": "~1.1",
     "generoi/genero-status": "*",
     "consolidation/robo": "^1.3",
     "generoi/robo-genero": "^0.2.2",
-    "wpackagist-plugin/jetpack": "^7.3"
+    "wpackagist-plugin/jetpack": "^7.3",
+    "wpackagist-plugin/gutenberg": "^5.9"
```
Comment (toffebjorkskog) [inline composer.json:103]: good to have included

File: composer.json [hunk 7]
```diff
@@ -106,7 +109,9 @@
     "brainmaestro/composer-git-hooks": "^2.2",
     "deployer/deployer": "^6.3",
     "deployer/recipes": "^6.1",
-    "generoi/deployer-genero": "^0.1-alpha"
+    "generoi/deployer-genero": "^0.1-alpha",
```
Comment (toffebjorkskog) [inline composer.json:112]: https://github.com/generoi/deployer-genero

File: composer.json [hunk 8]
```diff
@@ -106,7 +109,9 @@
...
+    "consolidation/robo": "^1.3",
+    "generoi/robo-genero": "^0.2.2"
```
Comment (toffebjorkskog) [inline composer.json:114]: https://github.com/generoi/robo-genero

File: web/app/mu-plugins/post-types.php [hunk 9]
```diff
@@ -89,6 +89,7 @@ public function registerPerson()
     {
         $person = new PostType('person', [
             'has_archive' => false,
+            'show_in_rest' => true,
```
Comment (toffebjorkskog) [inline web/app/mu-plugins/post-types.php:92]: Nice detail

**Comment by kimlehtinen** [comment]
> Other than that I'm fixing slow NFS syncs that break livereload

Great

File: web/app/mu-plugins/acf-json.php [hunk 10]
```diff
@@ -9,6 +9,12 @@
 License:      MIT License
 */
 
+namespace Genero\Site;
+
+if (!is_blog_installed()) {
+    return;
+}
+
```
Comment (toffebjorkskog) [inline web/app/mu-plugins/acf-json.php:17]: "doing things" :)

**Review by kimlehtinen — COMMENTED** [review]
If we have time for it next dev meeting, we could briefly go through what new plugins have been added and why

File: config/vagrant.config.yml [hunk 11]
```diff
@@ -9,6 +9,7 @@ vagrant_synced_folders:
     destination: /var/www/wordpress
     type: nfs
     create: true
+    mount_options: ['nolock', 'noatime', 'actimeo=1']
```
Comment (toffebjorkskog) [inline config/vagrant.config.yml:12]: Have you noticed any speedups.

**Review by toffebjorkskog — COMMENTED** [review]
Great work

File: config/vagrant.config.yml (hunk 11 above)
Comment (toffebjorkskog) [reply config/vagrant.config.yml:12]: This is for inital mounting right, not syncing?
or do you not `vagrant rsync-auto` anymore?

Comment (toffebjorkskog) [reply config/vagrant.config.yml:12]: > Other than that I'm fixing slow NFS syncs that break livereload. These could require more research but for now these have been working quite well. I have some more notes in the commit message.

i guess that answers my own question...

File: composer.json (hunk 3 above)
Comment (oxyc) [reply composer.json:90]: Spent about 2h trying to debug why users couldn't oEmbed youtube videos from time to time. Turned out it was JNT that had some weird firewall thing that made the curl requests take 0.6s and hit the timeout limit. No error was triggered anywhere except with this plugin.

Comment (oxyc) [reply composer.json:90]: Just useful to have in the codebase the day you're debugging something with network.

File: web/app/mu-plugins/post-types.php (hunk 9 above)
Comment (oxyc) [reply web/app/mu-plugins/post-types.php:92]: This is required for the Block Editor.

File: config/vagrant.config.yml (hunk 11 above)
Comment (oxyc) [reply config/vagrant.config.yml:12]: > see fideloper/Vaprobash#368 (comment)
>
> we dont force udp as it's in fact slower nowadays and not available on
linux. 

The TCP vs UDP performance differences I measured with Drupal VM a while back https://github.com/geerlingguy/drupal-vm/pull/1482#issuecomment-316821259

**Pull request merged**